dialogflow/stopwatch/Stopwatch/TokenReceiver.swift

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import Firebase
+
+extension Constants {
```

Comment:
Is `Constants` a name we control or an iOS standard/construct?

Reply:
Constants is struct defined by us to manage string literals.